tests/unit/test_utils.py

```diff
@@ -54,3 +56,8 @@ def test_add_years():
     assert tow_years_later.month == 3 and tow_years_later.day == 1
     assert four_years_later.month == initial_date.month and \
         four_years_later.day == initial_date.day
+
+
+def test_get_schema_for_resources(app):
+    """Test get_schemas_for_resource function."""
+    assert get_schema_for_resource(Patron) == get_schema_for_resource('ptrn')
```

Comment:
Maybe this test has not the expected effect if both functions calls responding None.
Would it be better to write two asserts like:
```
assert get_schema_for_resource(Patron) == 'https://ils.rero.ch/...'
assert get_schema_for_resource('ptrn') ==  'https://ils.rero.ch/...'
```

Reply:
sure I can do that ;-)
Just a little bit time consuming if (one day) we change the json schema of `Patron` resource. 
Done